<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Line3DSymbolWidget</class>
 <widget class="QWidget" name="Line3DSymbolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="labelHeight">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QgsDoubleSpinBox" name="spinHeight">
       <property name="minimum">
        <double>-99999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelExtrusion">
       <property name="text">
        <string>Extrusion</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QgsDoubleSpinBox" name="spinExtrusion">
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelClamping">
       <property name="text">
        <string>Altitude clamping</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cboAltClamping"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelBinding">
       <property name="text">
        <string>Altitude binding</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="cboAltBinding">
       <item>
        <property name="text">
         <string>Vertex</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Centroid</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelWidth">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QgsDoubleSpinBox" name="spinWidth">
       <property name="maximum">
        <double>99999.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="chkSimpleLines">
     <property name="text">
      <string>Render as simple 3D lines</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsCollapsibleGroupBox" name="groupShading">
     <property name="title">
      <string>Shading</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="3" column="0" colspan="2">
       <widget class="QgsMaterialWidget" name="widgetMaterial" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMaterialWidget</class>
   <extends>QWidget</extends>
   <header>qgsmaterialwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spinWidth</tabstop>
  <tabstop>spinHeight</tabstop>
  <tabstop>spinExtrusion</tabstop>
  <tabstop>cboAltClamping</tabstop>
  <tabstop>cboAltBinding</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
